### ISSUE TITLE
Convert ipython notebook to pdf and/or print notebook

### ISSUE THREAD

**Issue by tmstout** [jupyter/nbviewer#279]
Hi can someone please instruct me on how to convert my notebook.ipynb to notebook.pdf and/or print my notebook? Thank you.

**Comment by koelah** [comment]
Hello, someone to guide me on notebook conversion.

**Comment by bollwyvl** [comment]
@koelah the nbconvert link is out of date, here's the updated one:

https://nbconvert.readthedocs.io/en/latest/

If you can run `jupyter notebook` command locally, you've already got the `jupyter nbconvert` command. Installing a working LaTeX environment is a bit more involved!

For reference: at present, there are no plans to do latex-based PDF rendering of notebooks on nbviewer at this time.

**Comment by SumanthLazarus** [comment]
I think I've found a decent solution as I was stuck with the same problem. For an aesthetically pleasing print-out of the Jupyter notebook w. embedded pictures and codes (.pdf format) for the usage of study and learning (as lecture slides), I recommend take a print-out using your web-browser (Chrome: Print=Ctrl+P).
 
**Outcome**: an aestheically pleasing document, containing all codes, pictures as embedded within a Jupyter notebook. 

*tl:dr* Avoid any conversion within the Jupyter notebook option; print straight out of web-browser.

**Comment by kirankhanal2007** [comment]
I was able to do converting from Jupyter notebook to Latex, and then to pdf. It preserves all the contents as they are. Here are the steps:
https://github.com/kirankhanal2007/Python-Examples/blob/master/Convert%20Jupyter%20Notebook%20to%20PDF.pdf

**Comment by goekce** [comment]
Note that Jupyter Notebook is able to export a PDF.

`File -> Download as -> PDF via LaTeX`
